dts/riscv/ite/it8xxx2-pinctrl-map.dtsi

```diff
@@ -222,11 +222,11 @@
 		drive-open-drain;
 		bias-pull-up;
 	};
-	kso16_gpc3_default: kso16_gpc3_default {
+	kso16_default: kso16_default {
```

Comment:
How about adding a both the new and old names as valid nodelabels?  This will ensure downstream users aren't broken.
```suggestion
	kso16_default: kso16_gpc3_default: kso16_default {
```

Reply:
Yes, that would accommodate both.